assets/js/atcb.js

```diff
@@ -542,6 +542,10 @@ function atcb_generate_google(data) {
   if (data['description'] != null && data['description'] != '') {
     url += '&details=' + encodeURIComponent(data['description']);
   }
+  if (data['recurrence'] != null && data['recurrence'] != '') {
+    url += '&recur=' + encodeURIComponent(data['recurrence']);
```

Comment:
Codacy has a fix for the issue: [Replace `'&recur='·+·encodeURIComponent(data['recurrence'` with `"&recur="·+·encodeURIComponent(data["recurrence"`](https://app.codacy.com/gh/jekuer/add-to-calendar-button/pullRequest?prid=9703945)
```suggestion
    url += "&recur=" + encodeURIComponent(data["recurrence"]);
```